### DISCUSSION TITLE
billinear filtering when using gliden64

### DISCUSSION THREAD

**Discussion by asa975605** [mupen64plus-ae/mupen64plus-ae#1147] [Q&A] [answered]
Is there a way to get rid of the billinear filtering when scaling the gliden64 plugin? There's no filter when using angrylion.

**Comment by fzurita** [comment]
If you are talking about texture filtering mode. It's not bilinear by default. By default it's 3-point, which a N64 specific filtering mode.

If you go to profiles, then emulation, copy a GlideN64 profile there that you want, you can change the texture filtering mode there. Make sure you select your new profile afterwards.

**Comment by asa975605** [comment]
![Screenshot_20230714_064648_M64Plus FZ](https://github.com/mupen64plus-ae/mupen64plus-ae/assets/83899892/6a53961d-de46-4727-bcb0-26f2b1ad1d90)
![Screenshot_20230714_064624_M64Plus FZ](https://github.com/mupen64plus-ae/mupen64plus-ae/assets/83899892/fb2440da-2c2c-45a0-bc8a-7af80383ccce)
Thanks for the quick reply. Heres what I mean gliden64 is has blurry edges while angrylion has defined pixel edges. This is not related to texture filtering as far as I know but the idea is somewhat similar. I usually prefer then nearest neighbor scaling because it preserves the ui.

**Reply by fzurita** [reply, marked as answer]
Ok, I see what you mean. Under settings then display, set your resolution to "native" or match your phone display resolution. Then create a custom GLideN64 emulation profile and set the "Native Resolution Factor" to 1x.

**Comment by asa975605** [comment]
Thank you! Works perfectly.
